numpy/random/mtrand/mtrand.pyx

```diff
@@ -4888,6 +4889,247 @@ cdef class RandomState:
 
         return multin
 
+    def _multivariate_hypergeometric1(self, ndarray colors,
```

Comment:
Could then pass `colors` as a pointer. Might be easier to be sure of type consistency that way.